scripts/benchmark.sh

```diff
@@ -42,23 +42,38 @@ then
     exit 1;
 fi
 
-for ec_type in rs-gf2n-v rs-gf2n-c rs-gf2n-fft rs-gf2n-fft-add rs-gfp-fft rs-fnt rs-nf4; do
-  for k in 5; do
-    for m in 2; do
-      for word_size in 1 2 4 8; do
-        ${bin} -e ${ec_type} -w ${word_size} -k ${k} -m ${m} -c ${chunk_size} -s ${sce_type} -g ${threads_nb} -f ${show_type}
-        show_type=0
-      done
+chunk_size=51200
+sce_type=enc_only
+# for rs-nf4
+ec_type=rs-nf4
+word_size=8
+for k in 16; do
```

Comment:
Do we plan to add more values for `k` later?
If not, we can remove the loop to use k=16 and save one level of indentation.

Same for l.61 and 74-75

Reply:
I'd like to keep this for loop to facilitate extra runs :)
In CI, I define only a typical value for test :)